pr2eus_moveit/euslisp/collision-object-publisher.l

```diff
@@ -81,7 +81,12 @@
          (send colobj :primitive_poses
                (append pose
                        (list
-                        (ros::coords->tf-pose (if relative-pose relative-pose (send obj :worldcoords))))))))
+                        (ros::coords->tf-pose (let ((coords (if relative-pose
+                                                             relative-pose
+                                                             (send obj :worldcoords))))
+                                               (let* ((hv (float-vector 0 0 (/ (height-of-cylinder obj) 2)))
```

Comment:
- Why do you use `let` and `let*`? Is it impossible to use `let*` once?
- Please add a comment about the difference between euslisp cylinder's origin and ROS cylinder's origin.